MdeModulePkg/Universal/SetupBrowserDxe/Setup.c

```diff
@@ -306,7 +306,13 @@ UiCopyMenuList (
     Link     = GetNextNode (CurrentMenuListHead, Link);
 
     NewMenuEntry = AllocateZeroPool (sizeof (FORM_ENTRY_INFO));

```

Comment:
I feel like the caller should have a status to know whether the copy actually worked or not.

I don't think it's something you necessarily need to solve, but worth consideringg.